### PR TITLE
[bugfix] Fix CLI drops controller URL path when querying logs

### PR DESCRIPTION
**Pull request by moluzhang**
…er with prefix router

---
This change is [Reviewable](https://reviewable.io/reviews/fission/fission/1201)

### CODE REVIEW

**Comment by life1347** [comment]
@moluzhang  Thanks for the PR! I will give a test and hope we can merge it soon.

File: pkg/fission-cli/logdb/influxdb.go [hunk 1]
```diff
@@ -144,7 +144,7 @@ func (influx InfluxDB) query(query influxdbClient.Query) (*influxdbClient.Respon
 	}
 	// connect to controller first, then controller will redirect our query command
 	// to influxdb and proxy back the db response.
-	queryURL.Path = fmt.Sprintf("/proxy/%s", INFLUXDB)
+	queryURL.Path = fmt.Sprintf("%s/proxy/%s", queryURL.Path, INFLUXDB)
 	req, err := http.NewRequest("POST", queryURL.String(), nil)
```
Comment (life1347) [inline pkg/fission-cli/logdb/influxdb.go]: Though it's not included in changes, would you mind to change POST into http.MethodPost?

File: pkg/fission-cli/logdb/influxdb.go [hunk 2]
```diff
@@ -144,7 +144,7 @@ func (influx InfluxDB) query(query influxdbClient.Query) (*influxdbClient.Respon
 	}
 	// connect to controller first, then controller will redirect our query command
 	// to influxdb and proxy back the db response.
-	queryURL.Path = fmt.Sprintf("/proxy/%s", INFLUXDB)
+	queryURL.Path = fmt.Sprintf("%s/proxy/%s", queryURL.Path, INFLUXDB)
```
Comment (life1347) [inline pkg/fission-cli/logdb/influxdb.go]: Please add `path.Clean(fmt.Sprintf.....` to clean the URL path.

File: pkg/fission-cli/logdb/influxdb.go (hunk 1 above)
Comment (moluzhang) [reply pkg/fission-cli/logdb/influxdb.go]: Okay

File: pkg/fission-cli/logdb/influxdb.go (hunk 2 above)
Comment (moluzhang) [reply pkg/fission-cli/logdb/influxdb.go]: Okay, i will recommit it, thanks!

**Pull request merged**